src/Tizen.Multimedia/AudioManager/AudioDucking.cs

```diff
@@ -0,0 +1,182 @@
+ */
+
+using System;
+using System.Diagnostics;
+
+namespace Tizen.Multimedia
+{
+    /// <summary>
+    /// Provides the ability to control audio ducking.
+    /// </summary>
+    /// <seealso cref="AudioManager"/>
+    /// <since_tizen> 6 </since_tizen>
+    public sealed class AudioDucking : IDisposable
+    {
+        private AudioDuckingHandle _handle;
+        private bool _disposed = false;
+        private Interop.AudioDucking.DuckingStateChangedCallback _duckingStateChangedCallback;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AudioDucking"/> class with <see cref="AudioStreamType"/>.
+        /// </summary>
+        /// <param name="targetType">The type of sound stream to be affected by this new instance.</param>
+        /// <exception cref="ArgumentException"><paramref name="targetType"/> is invalid.</exception>
+        /// <exception cref="InvalidOperationException">Operation failed; internal error.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public AudioDucking(AudioStreamType targetType)
+        {
+            ValidationUtil.ValidateEnum(typeof(AudioStreamType), targetType, nameof(targetType));
+
+            _duckingStateChangedCallback = (AudioDuckingHandle ducking, bool isDucked, IntPtr _) =>
+            {
+                DuckingStateChanged?.Invoke(this,
+                    new AudioDuckingStateChangedEventArgs(IsDucked));
+            };
+
+            Interop.AudioDucking.Create(targetType, _duckingStateChangedCallback,
+                IntPtr.Zero, out _handle).ThrowIfError("Unable to create stream ducking");
+
+            Debug.Assert(_handle != null);
+        }
+
+        /// <summary>
+        /// Occurs when the ducking state is changed.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public event EventHandler<AudioDuckingStateChangedEventArgs> DuckingStateChanged;
+
+        /// <summary>
+        /// Gets the ducking state of the stream.
+        /// </summary>
+        /// <value>true if the audio stream is ducked; otherwise, false.</value>
+        /// <exception cref="InvalidOperationException">Operation failed; internal error.</exception>
+        /// <exception cref="ObjectDisposedException">The <see cref="AudioDucking"/> has already been disposed of.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public bool IsDucked
+        {
+            get
+            {
+                if (_disposed)
+                {
+                    throw new ObjectDisposedException(nameof(AudioDucking));
+                }
+
+                Interop.AudioDucking.IsDucked(Handle, out bool isDucked).
+                    ThrowIfError("Failed to get the running state of the device");
+
+                return isDucked;
+            }
+        }
+
+        /// <summary>
+        /// Activate audio ducking
+        /// </summary>
+        /// <param name="duration">The duration(milisecond) for ducking.</param>
```

Comment:
Suggestion:

        /// <param name="duration">The duration for ducking in milliseconds</param>